tests/test_regression.py

```diff
@@ -133,17 +135,18 @@ def test_sbottom_regionB_1400_550_60(
     CLs_obs, CLs_exp = calculate_CLs(
         sbottom_regionB_bkgonly_json, sbottom_regionB_1400_550_60_patch_json
     )
-    assert CLs_obs == pytest.approx(0.9744675266677597, rel=1e-5)
+    assert CLs_obs == pytest.approx(0.9944320696159622, rel=1e-5)
+
     assert np.all(
         np.isclose(
             np.array(CLs_exp),
             np.array(
                 [
-                    0.9338879894557114,
-                    0.9569045303300702,
-                    0.9771296335437559,
-                    0.9916370124133669,
-                    0.9983701133999316,
+                    0.9939882032779436,
+                    0.9961316425804301,
+                    0.9979736724671936,
+                    0.9992686756878328,
+                    0.9998593342137401,
```

Comment:
These changes look much larger than the changes in the surrounding regions, what could cause this?

Reply:
Yeah...I'll need to look at this again once I rebase it after we get another PR #1068 resolved.